.travis.yml

```diff
@@ -3,34 +3,30 @@ sudo: required
 dist: trusty
 
 env:
-    - LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/dependencies/libcsv/.libs JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 CLASSPATH=$TRAVIS_BUILD_DIR/dependencies/htsjdk-2.5.0.jar:$TRAVIS_BUILD_DIR/bin/genomicsdb-0.4.0-jar-with-dependencies.jar:$TRAVIS_BUILD_DIR/tests:$TRAVIS_BUILD_DIR/dependencies/json-simple-1.1.1.jar:.
+    - LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/dependencies/libcsv/.libs JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 CLASSPATH=$TRAVIS_BUILD_DIR/bin/genomicsdb-0.4.0-jar-with-dependencies.jar:$TRAVIS_BUILD_DIR/tests:.
 
 install:
     #Install dependencies
-    - sudo apt-get -y install lcov mpich zlib1g-dev libssl-dev
+    - sudo apt-get -y install lcov mpich zlib1g-dev libssl-dev rsync
     - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
     - sudo add-apt-repository -y ppa:openjdk-r/ppa
     - sudo apt-get update -q
     - sudo apt-get install g++-4.9 -y
     - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.9 60
-    - wget https://github.com/google/protobuf/archive/3.0.x.zip -O protobuf-3.0.x.zip
-    - unzip protobuf-3.0.x.zip
-    - pushd protobuf-3.0.x && ./autogen.sh && ./configure --prefix=/usr && make -j4 && sudo make install -j4 && popd
     - sudo apt-get -y install openjdk-8-jdk icedtea-plugin
     - jdk_switcher use openjdk8
     - echo $JAVA_HOME
     - export PATH=$JAVA_HOME/bin:$PATH
     - which javac && javac -version
     - which java && java -version
+    #Copy protobuf binaries
+    - wget https://github.com/Intel-HLS/GenomicsDB/releases/download/0.4.0/protobuf-3.0.2-trusty.tar.gz -O protobuf-3.0.2-trusty.tar.gz
+    - tar xzf protobuf-3.0.2-trusty.tar.gz && sudo rsync -a protobuf-3.0.2-trusty/ /usr/
```

Comment:
what I do in the gatk4 branch is 'export PROTOBUF_LIBRARY=$TRAVIS_BUILD_DIR/protobuf-3.0.2-trusty". This also works.